spec/factories/courses.rb

```diff
@@ -1,9 +1,11 @@
 FactoryGirl.define do
   factory :course do
-    title "MyCourse"
-    description "Course Details"
-    price 50.00
-    image "https://robohash.org/0.png"
-    category
+    sequence :title do |i|
```

Comment:
@yoasyo25 This is a great solution, we should explore sequencing all the factories. Thanks.